ratatui-widgets/src/block.rs

```diff
@@ -360,16 +471,18 @@ impl<'a> Block<'a> {
     ///
     /// # Example
     ///
-    /// This example positions all titles on the bottom except the "top" title which explicitly sets
-    /// [`TitlePosition::Top`].
+    /// This example positions all titles on the bottom by default. The "top" title explicitly sets
+    /// its position to `Top`, so it is not affected. The "foo" and "bar" titles will be positioned
+    /// at the bottom.
+    ///
     /// ```
     /// use ratatui::widgets::{Block, TitlePosition};
     ///
-    /// Block::new()
+    /// Block::bordered()
     ///     .title_position(TitlePosition::Bottom)
-    ///     .title_top("top")
-    ///     .title("foo")
-    ///     .title("bar");
+    ///     .title("foo") // will be at the bottom
+    ///     .title("bar") // will be at the bottom
+    ///     .title_top("top"); // will be at the top
```

Comment:
nit: changing order may demonstrate it better than calling `.title(...)` two times in a row
```suggestion
    ///     .title("foo") // will be at the bottom
    ///     .title_top("top"); // will be at the top
    ///     .title("bar") // will be at the bottom
```